src/Client/Web/Bit.Client.Web.BlazorUI.Tests/Buttons/BitButtonTests.cs

```diff
@@ -46,6 +46,28 @@ public Task BitButtonTest(Visual visual, bool isEnabled, ButtonStyle style)
             return Task.CompletedTask;
         }
 
+        [DataTestMethod,
+            DataRow(true, ButtonStyle.Primary, false, false),
+            DataRow(true, ButtonStyle.Standard, true, false),
+            DataRow(false, ButtonStyle.Primary, false, true),
+            DataRow(false, ButtonStyle.Standard, true, false),
+        ]
+        public Task BitButtonDisabledFocusTest(bool isEnabled, ButtonStyle style, bool allowDisabledFocus, bool expectedResult)
+        {
+            var com = RenderComponent<BitButtonTest>(parameters =>
+            {
+                parameters.Add(p => p.IsEnabled, isEnabled);
+                parameters.Add(p => p.ButtonStyle, style);
+                parameters.Add(p => p.AllowDisabledFocus, allowDisabledFocus);
+            });
+
+            var bitButton = com.Find(".bit-btn");
+
+            Assert.AreEqual(bitButton.HasAttribute("tabindex"), expectedResult);
```

Comment:
please add a check to the value of the "tabindex" attribute too.
Its value should be equal to "-1".